### PR TITLE
Remove accidental only and fix summarizer tests

### PR DESCRIPTION
**Pull request by kian-thompson**
Caused by #10179 and #10849

### CODE REVIEW

File: packages/runtime/container-runtime/src/opProperties.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const opSize = (op: ISequencedDocumentMessage): number => {
         op.contents :
         JSON.stringify(op.contents);
     const data = opHasData(op) ? op.data : "";
-    return content.length + data.length;
+    return content?.length + data.length;
```
Comment (markfields) [inline packages/runtime/container-runtime/src/opProperties.ts:15]: What does undefined + a number give? JS is notorious for doing weird stuff in cases like that I would be more explicit about that case

**Review by markfields — APPROVED** [review]
See my comment. Up to you if you change it though.

File: packages/runtime/container-runtime/src/opProperties.ts [hunk 2]
```diff
@@ -8,9 +8,12 @@ import { ISequencedDocumentMessage, ISequencedDocumentSystemMessage } from "@flu
...
     // Some messages may already have string contents,
     // so stringifying them again will add inaccurate overhead.
-    const content = typeof op.contents === "string" ?
-        op.contents :
-        JSON.stringify(op.contents);
+    let content = "";
+    if (op.contents) {
+        content = typeof op.contents === "string" ?
+            op.contents :
+            JSON.stringify(op.contents);
```
Comment (andre4i) [inline packages/runtime/container-runtime/src/opProperties.ts:15]:
```suggestion
            JSON.stringify(op.contents) ?? "";
```

and this way you can drop the `let`

**Review by andre4i — APPROVED** [review]

File: packages/runtime/container-runtime/src/opProperties.ts (hunk 1 above)
Comment (andre4i) [reply packages/runtime/container-runtime/src/opProperties.ts]: NaN

File: packages/runtime/container-runtime/src/opProperties.ts (hunk 2 above)
Comment (andre4i) [reply packages/runtime/container-runtime/src/opProperties.ts:15]: Currently, if JSON.stringify returns undefined, it will still override content.

**Pull request merged**